src/lucky/pretty_log_formatter.cr

```diff
@@ -54,6 +54,12 @@ struct Lucky::PrettyLogFormatter < Dexter::BaseFormatter
         arrow.colorize.dim
       end
     end
+
+    private def add_request_id : Nil
+      if id = local_context["request_id"]
+        io << " (#{id.colorize.dim})"
```

Comment:
What would be awesome here is if we could colorize the `id` to be a different color each time it changed... So when you look at the terminal, you see all the blue IDs or all the red ones and can easily tell which goes where... but I have no clue how we can do that unless the color was set much much higher and globally

Reply:
The color is an interesting idea. Here's one way to accomplish that. At first spin I had to pull in `big` -- I feel like this could also just use i64 by splicing the uuid in some way before converting it to an int.

```crystal
require "uuid"
require "big"
require "colorize"

uuid = UUID.random
hex = uuid.hexstring.to_big_i(16) 
color = (hex % 256).to_u8
puts "uuid: #{uuid.to_s.colorize(Colorize::Color256.new(color))}"
```

Reply:
oh snap! That's awesome 😂  I'm gonna try that.

Reply:
Yup! Looks like `hex = uuid.hexstring[0..6].to_i64(16)` works too. Not sure if the slice needs to be larger or not, but that seems to work...

Reply:
You're certainly discarding a lot of the entropy...which would be a problem if this was about anything more than just colors on a screen 😆 :shipit:

Reply:
Just realized that we can't guarantee the `id` will be UUID. With this setup, you have the ability to make the request ID anything you want as long as it's a string. It was a nice idea though 😂